internal/gps/source_manager.go

```diff
@@ -142,8 +163,9 @@ var _ SourceManager = &SourceMgr{}
 
 // SourceManagerConfig holds configuration information for creating SourceMgrs.
 type SourceManagerConfig struct {
-	Cachedir string      // Where to store local instances of upstream sources.
-	Logger   *log.Logger // Optional info/warn logger. Discards if nil.
+	Cachedir       string      // Where to store local instances of upstream sources.
+	Logger         *log.Logger // Optional info/warn logger. Discards if nil.
+	DisableLocking bool
```

Comment:
In keeping with the other items, let's have an inline explanatory comment.

Reply:
Okay -- will try to address these issues tonight.  Thank you!